satpy/tests/reader_tests/test_viirs_l1b.py

```diff
@@ -338,6 +338,27 @@ def test_load_dnb_angles(self):
             assert v.attrs["area"].lats.attrs["rows_per_scan"] == 2
             assert v.attrs["sensor"] == "viirs"
 
+    def test_scale_factor_and_offset_update(self):
+        """Test if scale_factor and add_offset are removed correctly."""
+        from satpy.readers import load_reader
+        from satpy.tests.utils import make_dataid
+        r = load_reader(self.reader_configs)
+        loadables = r.select_files_from_pathnames([
+            "VL1BM_snpp_d20161130_t012400_c20161130054822.nc",
+            "VGEOM_snpp_d20161130_t012400_c20161130054822.nc",
+        ])
+        r.create_filehandlers(loadables)
+        datasets = r.load(["M01",
+                           "M12",
+                           "I04",
+                           make_dataid(name="M01", calibration="radiance"),
+                           make_dataid(name="M12", calibration="radiance"),
+                           make_dataid(name="I04", calibration="radiance"),
+                           ])
+        for v in datasets.values():
+            assert ("scale_factor" not in v.attrs)
+            assert ("add_offset" not in v.attrs)
```

Comment:
Couldn't these checks be added to another existing test rather than have a whole new test just for attribute checks?